plugins/poll/spec/controllers/posts_controller_spec.rb

```diff
@@ -191,26 +188,14 @@
 
         describe "with no vote" do
 
-          it "OP can change the options" do
+          it "can change the options" do
             put :update, params: {
               id: post_id, post: { raw: new_option }
             }, format: :json
 
             expect(response.status).to eq(200)
             json = ::JSON.parse(response.body)
-            expect(json["post"]["polls"]["poll"]["options"][1]["html"]).to eq("C")
-          end
-
-          it "staff can change the options" do
```

Comment:
I think we still need this test case to ensure that staff members are able to edit options.

Reply:
My bad I just read that we simplified the update logic. Is there any previous context of why we do so?